security/pkg/adapter/vault/ca.go

```diff
@@ -17,11 +17,31 @@ package vault
 
 import (
 	"fmt"
+	"io/ioutil"
 	"time"
 
+	"github.com/hashicorp/vault/api"
+
 	"istio.io/istio/security/pkg/pki/util"
 )
 
+// Config for prototyping purpose
+const (
+	istioCaMountPoint   = "istio_ca"
+	istioCaDescription  = "Istio CA"
+	configCaKeyCertPath = "istio_ca/config/ca"
+	workloadRolePath    = "istio_ca/roles/workload_role"
+	signCsrPath         = "istio_ca/sign-verbatim"
+)
+
+// Config for prototyping purpose
+const (
+	vaultAddrForTesting = "http://127.0.0.1:8200"
+	tokenForTesting     = "myroot"
+	testCAKeyCertFile   = "testdata/istio_ca.pem"
```

Comment:
It's OK to put it here for a prototype. This can be moved to ca_test.go in the following PRs.

Reply:
+1 to move to xx_test.go, could you leave a TODO?